plugins/labhub.py

```diff
@@ -201,3 +201,62 @@ def mark_cmd(self, msg, match):
                         'so you will get feedback from the community. Use '
                         '`cobot mark wip` if there are known issues that should'
                         ' be corrected by the author.'.format(mr_link=mr.url))
+
+    @re_botcmd(pattern=r'^assign\s+https://(github|gitlab)\.com/([^/]+)/([^/]+)/issues/(\d+)',  # Ignore LineLengthBear, PyCodeStyleBear
+               flags=re.IGNORECASE)
+    def assign_cmd(self, msg, match):
+        """Assign to GitLab and GitHub issues."""  # Ignore QuotesBear
+        org = match.group(2)
+        repo_name = match.group(3)
+        iss_number = match.group(4)
+
+        user = msg.frm.nick
+
+        try:
+            assert org == self.GH_ORG_NAME or org == self.GL_ORG_NAME
+        except AssertionError:
+            yield 'Repository not owned by our org.'
+            return
+
+        checks = [
```

Comment:
I would rather you just create a separate method. If you want to add a method to a list to use the for loop in eligible use a decorator to do so.
```
checks = []


def register_check(function):
    checks.append(function)

    return function

@register_check
def function_to_add_to_check():
    """
    I would like a docstring for the check.
    """
```